test/reducer.test.js

```diff
@@ -383,3 +406,11 @@ describe('networkSelector', () => {
     });
   });
 });
+
+describe('network reducer config', () => {
```

Comment:
What's the purpose of this test?

Reply:
I think this is a leftover from previous code, I'll remove it. Good catch :+1: